components/runners/ambient-runner/tests/test_grpc_transport.py

```
@@ -0,0 +1,474 @@
+            except asyncio.CancelledError:
+                pass
+
+    async def test_user_message_triggers_bridge_run(self):
+        payload = _make_runner_payload(thread_id="t-1", run_id="r-1")
+        msgs = [_make_session_message("user", payload, seq=1)]
+        client = _make_grpc_client(messages=msgs)
+        bridge = _make_bridge()
+
+        events = [make_text_start(), make_text_content(), make_run_finished()]
+
+        async def fake_run(input_data):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `fake_run`

(ANN202)